compysition/actor.py

```diff
@@ -206,9 +205,12 @@ def _loop_send(self, event, queues, check_output=True):
         if check_output and not isinstance(event, self.output):
             raise InvalidActorOutput("Event was of type '{_type}', expected '{output}'".format(_type=type(event), output=self.output))
 
-        if len(queues) > 0:
-            self._send(queues[0], deepcopy(event))
-            map(lambda _queue: self._send(_queue, deepcopy(event)), queues[1:])
+        if isinstance(queues, _InternalQueuePool):
```

Comment:
Why not? What if you extended `_InternalQueuePool` and wanted to inject another dependency so to speak. Python's MRO is not linear. `isinstance` in in the language for a reason. It's use case is just rare enough for me to wonder if try catch would be more flexible and idiomatic. I admit I do not have all the background.